osquery/events/linux/syslog.h

```diff
@@ -0,0 +1,172 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include <fstream>
+#include <stdio.h>
+#include <map>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include <gtest/gtest.h>
```

Comment:
Please do not include gtest here. `osquery/core.h` includes the test class name interpolation macro: https://github.com/facebook/osquery/blob/master/include/osquery/core.h#L32